catalog/ui/search-ui/search-ui-app/pom.xml

```diff
@@ -87,6 +86,23 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.karaf.tooling</groupId>
+                <artifactId>features-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>create-kar</id>
+                        <goals>
+                            <goal>create-kar</goal>
+                        </goals>
+                        <configuration>
+                            <!-- Workaround to prevent features.xml file from being included twice in the kar file. -->
```

Comment:
It's not entirely clear to me what causes the double include, but pretty sure you could fix it by adjusting the phases (ordering) of some of the plugins, instead of this. Would look cleaner, at least.